src/commands/CommandMusicPlay.java

```diff
@@ -19,100 +19,110 @@
+							
+							YouTube youtube = new YouTube.Builder(
+									new NetHttpTransport(),
+									new JacksonFactory(),
+									new HttpRequestInitializer(){
+										public void initialize(
+												HttpRequest request)
+												throws IOException{}
+									}).setApplicationName("Discord Bot")
+									.build();
+							
+							YouTube.Search.List search = youtube.search().list(
+									"snippet");
+							
+							search.setMaxResults((long)1);
+							search.setQ(getContent());
+							search.setKey(Environment.getVar("YOUTUBE_TOKEN"));
+							
+							SearchListResponse response = search.execute();
+							
+							id = response.getItems().get(0).getId()
+									.getVideoId();
+							
+							source = "https://www.youtube.com/watch?v=" + id;
+							
+						}
+						else{
+							source = getContent();
+						}
+						
+						MusicManager.get().loadTrack(this, source);
+						
+					}
+					catch(IOException e){
+						
+						sendMessage(lang("SongByStringFail"));
 						
-						YouTube youtube = new YouTube.Builder(new NetHttpTransport(), new JacksonFactory(), new HttpRequestInitializer() {
-							public void initialize(HttpRequest request) throws IOException {
-							}
-						}).setApplicationName("Discord Bot").build();
-
-						YouTube.Search.List search = youtube.search().list("snippet");
-
-						search.setMaxResults((long) 1);
-						search.setQ(getContent());
-						search.setKey(Environment.getVar("YOUTUBE_TOKEN"));
-
-						SearchListResponse response = search.execute();
-
-						id = response.getItems().get(0).getId().getVideoId();
-
-						source = "https://www.youtube.com/watch?v=" + id;
-
-					} else {
-						source = getContent();
 					}
 					
-					MusicManager.get().loadTrack(this, source);
-				
-				} catch (IOException e) {
-				
-					sendMessage(lang("SongByStringFail"));
-				
 				}
-
-			} else {
-
-				MusicPlayer player = MusicManager.get().getPlayer(this);
-
-				if (player.isPaused()) {
-					player.setPause(false);
-
-					sendMessage(lang("Resuming"));
-				} else {
-					new BotError(this, lang("NoContent"));
+				else{
+					
+					MusicPlayer player = MusicManager.get().getPlayer(this);
+					
+					if(player.isPaused()){
+						player.setPause(false);
+						
+						sendMessage(lang("Resuming"));
+					}
+					else{
+						new BotError(this, lang("NoContent"));
+					}
+					
 				}
-
+				
 			}
-
 		}
-
+		
 	}
-
+	
 	@Override
-	public Object getCalls() {
+	public Object getCalls(){
 		return MUSIC_PLAY;
 	}
-
+	
 	@Override
-	public String getCommandDescription() {
+	public String getCommandDescription(){
 		return "Start a song by giving a youtube link or restart a paused song by not giving a link";
-
+		
 	}
-
-	public boolean isUrl(String string) {
-		try {
+	
+	public boolean isUrl(String string){
+		try{
 			URL url = new URL(string);
 			return true;
-		} catch (MalformedURLException e) {
+		}
+		catch(MalformedURLException e){
 			return false;
 		}
 	}
```

Comment:
You are testing for a parameter but you are not implementing the method `getParameterHelp` to let the user know there is a `l` parameter.

Also, you'll be able to link the parameter `l` and `latest` together using this.